src/main/java/com/epam/reportportal/jobs/clean/CleanStorageJob.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.reportportal.jobs.clean;
+
+import com.epam.reportportal.jobs.BaseJob;
+import com.epam.reportportal.storage.DataStorageService;
+import com.epam.reportportal.storage.MinioDataStorageService;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Removing data from storage.
+ */
+@Component
+public class CleanStorageJob extends BaseJob {
+
+    private final DataStorageService storageService;
+    private static final int CHUNK_SIZE = 1000;
+
+    public CleanStorageJob(JdbcTemplate jdbcTemplate, DataStorageService storageService) {
+        super(jdbcTemplate);
+        this.storageService = storageService;
+    }
+
+    @Scheduled(cron = "${rp.environment.variable.clean.storage.cron}")
+    @SchedulerLock(name = "cleanStorage", lockAtMostFor = "24h")
+    public void execute() {
+        logStart();
+        AtomicInteger counter = new AtomicInteger(0);
+
+        String chunkDataQuery = "DELETE FROM attachment_deletion WHERE id in " +
```

Comment:
Is it a good practice to store string as a local variable and use concatenation  by +? 
It can be provided as a query parameter

Reply:
As for me, for this kind of situations readability of code if more important, of course we can remove string "sum" to avoid little overhead, and need to be checked is it possible to set param for limit expression.